[kirkstone] pkggrp-ni-internal-deps: add PXIPS ATS deps

amstewart opened the pull request:
Per [NI AZDO 2463687](https://dev.azure.com/ni/DevCentral/_boards/board/t/RTOS/Work%20Items/?workitem=2463687), the PXI PS ATS depends on the `memtester` package.

Express this dependency in the internal-deps packagegroup, to ensure that it is built and included in the core feed.

# Testing
* [x] Built `packagegroup-ni-internal-deps` with this change; validated that `memtester` is built as a dependency.

amstewart committed:
pkggrp-ni-internal-deps: add PXIPS ATS deps

Signed-off-by: Alex Stewart <alex.stewart@ni.com>

gratian commented on recipes-core/packagegroups/packagegroup-ni-internal-deps.bb:
```diff
@@ -87,3 +87,10 @@ RDEPENDS:${PN} += "\
+# Team: PXI PS
+# Contact: Kevin Khai-Wern Lim
+RDEPENDS:${PN} += "\
+	memtester \
```
Do we need to remove `memtester` from [packagefeed-ni-extra.bb](https://github.com/ni/meta-nilrt/blob/7ffc454b3fab7c3ee5c4d3aff119a0751f6e63d5/recipes-core/packagegroups/packagefeed-ni-extra.bb#L235) since we're adding it here?

amstewart replied:
No. It should be automatically deconflicted by the build scripting.

gratian replied:
I get that the build tooling can de-conflict it but from a maintenance perspective it seems odd to me to have it both in `packagefeed-ni-extra.bb` and `packagefeed-ni-core.bb` (via packagegroup-ni-internal-deps). In the past IIRC when we moved things into internal deps we removed it from extra.

amstewart replied:
You're right that people do that, but it has never been necessary. I prefer to be conservative wrt. ensuring that needed deps get built into **a** feed, and avoid the possibility that something gets accidentally dropped from all he feeds.

amstewart replied:
Will fix.

amstewart committed:
pkgfeed-ni-extra: remove memtester dep

memtester was added to the extras feed without justification, and it is
already being included in the core feed as a dependency of the PXI PS
ATS from pkggrp-ni-internal-deps.

Remove its duplicate dependency for cleanliness.

Signed-off-by: Alex Stewart <alex.stewart@ni.com>
